tests/Tokenizer/TokensTest.php

```diff
@@ -1002,6 +1002,39 @@ public function provideEnsureWhitespaceAtIndexCases()
         ];
     }
 
+    public function testAssertTokensAfterChanging()
```

Comment:
I removed and re-added this. The test covers a part of `AssertTokensTrait` which is not available on 2.2 (introduced on master only)